opennms-web-api/src/main/java/org/opennms/web/svclayer/support/DefaultAdminApplicationService.java

```diff
@@ -307,12 +325,25 @@ public void performEditLocations(final String applicationIdString, final String
         }
     }
 
+    private void sentEvent(final OnmsApplication application, final String uei) {
```

Comment:
Better call it sendEvent, 'sent' is past.

Reply:
:+1: